### DISCUSSION TITLE
Cannot initialize metadata

### DISCUSSION THREAD

**Discussion by dannybirdi** [tm01013/how-to-make-your-own-crypto#9] [Testing] [answered]
Hi, I'm getting an error when trying to initialize my tokens.
C:\>spl-token initialize-metadata 3A*** 'mytokenname' 'tokensymbol' 'https://api.npoint.io/***' -v -p TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb
error: Found argument 'initialize-metadata' which wasn't expected, or isn't valid in this context

**Comment by tm01013** [comment, marked as answer]
Hi, It seems you not installed the solana cli properly. Try reinstalling, or use a linux or unix machine.
https://spl.solana.com/token#setup https://docs.solanalabs.com/cli/install
